krux_sqs/sqs.py

```diff
@@ -219,10 +220,17 @@ def send_messages(self, queue_name, messages):
                 else:
                     raise TypeError('Message must be either a dictionary or a string')
 
-                entries.append({
-                    'Id': Sqs._get_random_id(),
-                    'MessageBody': msg,
-                })
+                if group_id is None:
+                    entries.append({
+                        'Id': Sqs._get_random_id(),
+                        'MessageBody': msg,
+                    })
+                else:
+                    entries.append({
+                        'Id': Sqs._get_random_id(),
+                        'MessageBody': msg,
+                        'MessageGroupId': group_id,
+                    })
```

Comment:
How about
```
msg = {
  'Id': Sqs.get_random_id()
  'MessageBody': msg
}
if group_id is not None:
  msg['MessageGroupId'] = group_id

entries.append(msg)
```
this will be slightly more dry.